wasmedge-ggml-llama-interactive/README.md

```diff
@@ -134,21 +134,33 @@ The total cost of 3 apples would be 15 dollars. Each apple costs 5 dollars, so 3
 
 ## Parameters
 
-Currently, we support the following parameters:
+Currently, we support setting llama options using `set_input` with index 1.
+You can pass the JSON string as a `Vec<u8>` type to `set_input`.
 
-- `LLAMA_LOG`: Set it to a non-empty value to enable logging.
-- `LLAMA_N_CTX`: Set the context size, the same as the `--ctx-size` parameter in llama.cpp (default: 512).
-- `LLAMA_N_PREDICT`: Set the number of tokens to predict, the same as the `--n-predict` parameter in llama.cpp (default: 512).
+Supported parameters include:
 
-These parameters can be set by adding the following environment variables before the `wasmedge` command:
+- `enable-log`: Set it to true to enable logging.
+- `stream-stdout`: Set it to true to print the inferred tokens to standard output.
+- `ctx-size`: Set the context size, the same as the `--ctx-size` parameter in llama.cpp.
+- `n-predict`: Set the number of tokens to predict, the same as the `--n-predict` parameter in llama.cpp.
+- `n-gpu-layers`: Set the number of layers to store in VRAM, the same as the `--n-gpu-layers` parameter in llama.cpp.
+- `reverse-prompt`: Set it to the token at which you want to halt the generation. Similar to the `--reverse-prompt` parameter in llama.cpp.
+- `batch-size`: Set the number of batch size for prompt processing, the same as the `--batch-size` parameter in llama.cpp.
```

Comment:
Is it possible to give the default values of the options or some of them? Thanks!

Reply:
Sure, I'll add it.

Reply:
Done.